imports/plugins/core/orders/client/components/DataTable/OrderTotalCell.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Box } from "@material-ui/core";
+
+/**
+ * @name OrderTotalCell
+ * @param {Object} row A react-table row object
+ * @return {React.Component} A React component to render an order's total
+ */
+function OrderTotalCell({ row }) {
+  // Sum up all the payments on an orders. Since the API returns
+  // the payments already formatted, just use the currency symbol provided.
+  const currencySymbol = row.original.payments[0].amount.displayAmount.charAt(0);
+  const reducer = (accumulator, value) => accumulator + Number.parseFloat(value.amount.displayAmount.slice(1));
+  const total = row.original.payments.reduce(reducer, 0);
```

Comment:
@willopez are you trying to show the order total or the total amount paid for the order. If it's the order total then you can get that from the `summary` object on the order.

Either way, the total should come from the API, rather than being calculated on the fly there.

Reply:
Tru story, now using order total provided by API